lib/generators/hayfork/templates/migrations/rebuild.rb

```diff
@@ -3,6 +3,6 @@
 class Rebuild<%= table_name.camelize %> < ActiveRecord::Migration<%= migration_version %>
   def up
     execute <%= class_name %>.triggers.replace
-    execute <%= class_name %>.triggers.rebuild.lines
+    execute <%= class_name %>.triggers.rebuild
```

Comment:
🤔 Builder does (or did) this — it worked fine. Did this change in a recent version of Rails?

Reply:
Thank you for the review!

There was an `execute` migration helper in Builder. Using `execute` in this generator, rather than `execute.lines`, makes this generator work out-of-the-box without that dependency.